packages/utils/scripts/compare_resource_counts_per_file.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+## This script pulls the per resource count reference file from s3 and compares it to the file passed in as an argument
+
+# Check if one argument is passed
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <file1>"
+  exit 1
+fi
+
+# Assign argument to a variable for better readability
+FILE1=$1
+
+# Fixed S3 path for file2
+S3_FILE2="s3://fhir-resource-count/develop-per-file-resource-counts.json"
```

Comment:
There's a bucket we can use for this, no need to create another one: `devs.metriport.com`

We can just create a new folder (with the name of the package to give better context, `fhir resource count` doesn't say much as to where its used.